test/e2e/operator_test.go

```diff
@@ -558,6 +662,27 @@ func TestUpdateDefaultIngressController(t *testing.T) {
 		t.Logf("secret content=%v", string(secret.Data["tls.crt"]))
 		t.Fatalf("failed to observe update of default ingress CA certificate configmap: %v\noriginal=%v\ncurrent=%v", err, previousDefaultIngressCAConfigmap.Data["ca-bundle.crt"], defaultIngressCAConfigmap.Data["ca-bundle.crt"])
 	}
+
+	// Verify that the router-certs secret gets updated.
+	previousRouterCertsSecret = routerCertsSecret.DeepCopy()
+	wait.PollImmediate(1*time.Second, 20*time.Second, func() (bool, error) {
```

Comment:
Timeout values look arbitrary (e.g. 10s, 15s, 20s). Would it make sense to use the same timeout values for consistency or to write a brief explanation of why a particular value is preferred?

Reply:
There was probably some off-the-cuff reasoning about how long it might take the operator to update a resource, or assumptions that if resource A has been updated, then resource B should have been updated too.  However, in lieu of detailed explanations, I've updated the polling loops all to use `timeout` for consistency.  This increases the timeouts for some polling loops; at worst, it will take a little longer for the test to fail, and at best, this can make the test less flaky.